### ISSUE TITLE
Unable to load with multiple wavesurfer instances

### ISSUE THREAD

**Issue by bkentdev** [katspaugh/wavesurfer.js#2003]
When trying to load a blob via `wavesurfer.loadBlob` I get the error:
```
Uncaught (in promise) DOMException: Unable to decode audio data
```

I have also converted the blob to a URL and tried using `wavesurfer.load` but get the same error.

The type on the blob is: `type: "audio/webm;codecs=opus"`
Wavesurfer.js version: 3.3.3

Also, I am mapping through a list of blobs with Stencil.js (React-like framework), and the first one loads just fine. It's the consecutive wavesurfer instances that throw the error. I am building the waveform container element using Stencil refs: `ref={(element) => this.waveReference = element}`, which are similar to React refs. Could be that because I have a bunch of identical elements that the wavesurfer container cannot instantiate properly after the first one? I have tried custom `id`s but that doesn't work so I'm not sure there.

Likely related to: https://github.com/katspaugh/wavesurfer.js/issues/703, and if so a workaround would be very helpful.

**Comment by thijstriemstra** [comment]
> Wavesurfer.js version: 3.3.3

Always test with newest version.

**Comment by bkentdev** [comment]
Updated to 4.0.1, still seeing issue.

**Comment by bkentdev** [comment]
Found the answer. When you are using MediaRecorder, as you record you save data in chunks. I was trying to render each chunk with Wavesurfer.js every 250ms in an effort to do a real-time waveform. MDN docs suggest that individual chunks may not be playable until you have finished recording and concat all chunks into one blob.

See "Note" in  [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/MediaStream_Recording_API#Overview_of_the_recording_process)

Tl;dr- not an issue with Wavesurfer.js, chunks in MediaRecorder are not playable until you are finished recording and combine them into a single blob file.
